proposals/drafts/disclosures.rst

```diff
@@ -0,0 +1,806 @@
+
+classification
+    The category of the disclosure type. Current values include:
+        
+    * lobbying      - Disclosures related to lobbying
+    * contributions - Disclosures related to political contributions
+
+amends_type
+    The identifier of the disclosure type that this disclosure type is able to amend. Can be the same as identifier, where future submissions supercede past submissions.
+
+amendment
+    **optional**
+    A boolean that is true if this is a registration type that is reserved for amending other registration types
+
+DisclosedEvent
+--------------
+The basis for the DisclosedEvent is the Open Civic Data ``Event`` type, as described in `OCDEP 4: Events <http://opencivicdata.readthedocs.org/en/latest/proposals/0004.html>`_. Constraints on field values specified below
+
+identifier
+    An identifier that uniquely identifies the event.
+
+classification
+    As defined in the ``Event`` type, where values are extended to include:
+
+    * lobbying
+    * contribution
+
+participants
+    Participants associated with the event. 
+
+    note
+        As defined on the ``Event`` type, where values identifies the role of the participant. Choices include the following, and may be updated:
+          * lobbyist      - (lobbying) person who actually did the lobbying
+          * lobbied       - (lobbying) organizations and/or individuals that were lobbied
+          * regarding     - (lobbying) bills, regulations or other matters that can be identified
+          * contributor   - (contributions) the source of the transaction
+          * recipient     - (contributions) the target of the contribution
+          * lender        - (contributions) the source of a loan
+          * borrower      - (contributions) the recipient of a loan
+          * creditor      - (contributions) entity to which a debt is owed
+          * debtor        - (contributions) entity which owes a debt
+
+Form
+----
+Object representing form used for making disclosures
+
+identifier
+    An identifier that uniquely identifies the form
+
+form_type
+    An identifier that points to the type of form. See ``FormType`` section below.
+
+fields
+    A list of sub-objects representing fields in the form. See ``Field`` section below.
+
+FormType
+~~~~~~~~
+The type of a form
+
+identifier
+    An identifier that uniquely identifies the form type
+
+name
+    The canonical name of the form
+
+description
+    description of the form
+
+Field
+-----
+Object representing a field used in a disclosure form
+
+identifier
+    An identifier that uniquely identifies the field
+
+field_type
+    An identifier that points to the type of field. See ``FieldType`` section below.
+
+value
+    The value of the field
+
+FieldType
+~~~~~~~~~
+The type of a field
+
+identifier
+    An identifier that uniquely identifies the field type
+
+name
+    The canonical name of the field
+
+description
+    Description of the field
+
+Schedule
```

Comment:
So actually I'm going to update the pull with the form/field and schedule/line stuff taken out. it's going to create too much overhead to have them in the OCDEP itself. i'd always meant for the DisclosureEvent object to be an abstraction over the actual filings, but wanted to be able to include the filings as well. after speaking to @paultag, the abstraction seems sufficient for the needs of OCD, and the rest would be more appropriate in the extras section, since it would potentially (almost certainly) vary widely across disclosure authorities.

FWIW, though, I think of schedules as a type of field that has its own internal structure and can expand to include an array of related records.  typically, it's associated with some summary number that is a field on a form (ie, a schedule of itemized contributions being the source of a "total contributions" field in some contributions form).  

the LD-2 is badly designed, and it's a great example of why fields and forms can't be this kind of object: the lobbying activity section should be accounted for with schedules rather than regular fields.  as it is currently, questions 15-19 need to be repeated an arbitrary number of times. eg: http://soprweb.senate.gov/index.cfm?event=getFilingDetails&filingID=b31714db-4fe5-4f5a-b154-be885d7942ea&filingTypeID=78

again, i'm removing all of this in the updated pull, so this is academic.